### ISSUE TITLE
BachelierNet Multichannel Scenario

### ISSUE THREAD

**Issue by payas-parab-92** [jankrepl/deepdow#75]
I was looking at the code for class BachelierNet(torch.nn.Module, Benchmark) at (https://github.com/jankrepl/deepdow/blob/master/deepdow/nn.py)

Doesn't this line (line 143) in the nn.py imply that the returns series is located at channel 0? I am wondering because the loss functions take an integer (returns_channel : int Which channel of the `y` target represents returns.). For the BachelierNet there is no such functionality and this can cause errors because that channel will be fed into the covariance matrix for the portfolio optimization layer. 

Great job on this library! Incredible stuff you got here.

**Comment by jankrepl** [comment]
Thank you for the question!

You are right, that for the `BachelierNet` the `returns_channel` is effectively hardcoded to 0. I guess I decided to do this on purpose since these networks already have a bunch of hyperparameters. However, if you want it to be changed, feel free to create a pull request!

Related to this, all the networks should be taken with a grain of salt. More than anything, they are in the source code to demonstrate the flexibility of the framework. I would definitely encourage you to write your own network that makes the most sense in your application!

**Comment by jankrepl** [comment]
I am going to close this issue because of inactivity. Feel free to reopen!
